File: Python/fibonacci_triangle.py
# Fibonacci triangle in Python #

n = int(input())
sum = 0
num1 = 0
num2 = 1
for i in range(1, n+1):
    for j in range(1, i+1):
        if i == 1 and j == 1:
            print(0)
        elif i == 2 and j == 1:
            print(1, end="\t")
        elif j != i:
            sum = num1 + num2
            print(sum, end="\t")
            num1 = num2
            num2 = sum
        else:
            sum = num1 + num2
            print(sum)
            num1 = num2
            num2 = sum



    

File: C/fibonacci_triangle.c
#include <stdio.h>

int main(void)
{
    int x;
    scanf("%d", &x);

    int num1 = 0;
    int num2 = 1;
    int sum = 0;
    for(int i = 1; i <= x; i++){
        for(int j = 1; j <= i; j++){
            if(i == 1 && j == 1){
                printf("0\t");
            }
            else if(i == 2 && j == 1){
                printf("1\t");
            }
            else{
                sum = num1 + num2;
                printf("%d\t", sum);
                num1 = num2;
                num2 = sum;
            }
        }
        printf("\n");
    }
    return 0;
}